AutoROM/licensed/checksums.txt

```diff
@@ -0,0 +1,108 @@
+4b27f5397c442d25f0c418ccdacf1926 adventure
```

Comment:
If you are going to add  this file can you remove the old one?

Reply:
Do you think we even need the checksum file? There's a guarantee the ROMs that are downloaded are packaged in such a way that they conform to this list?

Reply:
I take that back, for the time being multi-agent ALE relies on the ROM name which is derived from the checksum map. I'm just going to hardcode this into `AutoROM.py` or else things become more complicated with this subpackage deal.